.buildkite/shared-pipeline-vars

```diff
@@ -4,8 +4,8 @@
 # to set up some variables that will be interpolated in the `.yml` pipeline before uploading it.
 
 # 🎗️ If you update the image to a newer Xcode version, don't forget to also update the badge in the README.md file accordingly for consistency
```

Comment:
I see that this was missed despite the reminder comment 😄

Reply:
@AliSoftware done!